common/beerocks/bcl/source/son/son_wireless_utils.cpp

```diff
@@ -832,3 +832,34 @@ beerocks::eBssType wireless_utils::wsc_to_bwl_bss_type(WSC::eWscVendorExtSubelem
 
     return beerocks::BSS_TYPE_INVALID;
 }
+
+std::list<uint8_t> string_to_wsc_oper_class(const std::string &operating_class)
+{
+    std::list<uint8_t> radio_24g = {81, 82, 83, 84};
+    std::list<uint8_t> radio_5g  = {115, 116, 117, 118, 119, 120, 121, 122,
+                                   123, 124, 125, 126, 127, 128, 129, 130};
```

Comment:
I think that you may do operating classes lists a global constants instead of local ones:
```
    const std::list<uint8_t> radio_24g{81, 82, 83, 84};
    const std::list<uint8_t> radio_5g{115, 116, 117, 118, 119, 120, 121, 122,
                                                        123, 124, 125, 126, 127, 128, 129, 130};
    const std::list<uint8_t> radio_24g_5g(std::list<uint8_t>(radio_24g).merge(radio_5g))
```

Reply:
@MuteSpirit Proposed code does not work.